strict_encoding/src/strategies.rs

```diff
@@ -48,6 +53,7 @@ pub trait Strategy {
     /// - [`BitcoinConsensus`]
     /// - [`Wrapped`]
     /// - [`UsingUniformAddr`]
+    /// - [`MoneroConsensus`]
```

Comment:
This will fail on docs compiling w/o monero feature. We need
```suggestion
    #[cfg_attr(feature = "monero", doc = "- [`MoneroConsensus`]")]
```